src/Assignment3/Employee.java

```diff
@@ -0,0 +1,139 @@
+package Assignment3;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.ListIterator;
+import java.util.Scanner;
+
+
+public class Employee implements Serializable {
+
+    long ID;
+    String name;
+    String email_address;
```

Comment:
You can use lower camel case even though this is also good but try to use the same pattern for all variable name.